src/UCTNode.h

```diff
@@ -79,16 +79,11 @@ class UCTNode {
     void link_nodelist(std::atomic<int>& nodecount,
                        std::vector<Network::scored_node>& nodelist,
                        float init_eval);
-
-    // Tree data
-    std::atomic<bool> m_has_children{false};
-    std::vector<node_ptr_t> m_children;
-
     // Move
-    int m_move;
+    int16_t m_move;
```

Comment:
Needs a std:: prefix as it's not guaranteed to be in the global namespace. Also perhaps the move type should be a typedef if we do this.